README.md

```diff
@@ -1,3 +1,32 @@
+News Summary challenge
+=================
+This app will grab all the headlines from the Guardian newspaper API and display them on a page.  Clicking on a headline will show a summary of the article.
+
+## Features
+- [x] See all of today's headlines (10) in one place
+- [x] Display a picture for each headline
+- [x] Click a headline to see a summary
+- [x] Click the summary headline to go to the original
+- [ ] Responsive design
+- [ ] Animations
+
```

Comment:
Omg! I will definitely steal the "todo list" idea for my README next time! 
Well done!